.gitignore

```diff
@@ -14,3 +14,5 @@ vetur.config.js
 beamer/data/relayers
 relayer/relayer-*-x64
 addresses.json
+
+node_modules
```

Comment:
I understand that `node_modules` is no longer used with yarn -- could you please clarify whether that is the case?
My impression is that yarn now stores cached packages under `.yarn`.

Reply:
All the node package managers use `node_modules` and yarn is not an exception.

Reply:
Yeah, I just assumed we're going to use yarn 2, which does not use `node_modules` anymore.